source/includes/examples/_find-the-center.html.md

```diff
@@ -0,0 +1,182 @@
+# Find the Center Example
+
+In this example, we'll walk you through the various statements that are part of the Find the Center project, including the simulator file and the Inkling file. Find the Middle was created as a very basic example of Inkling and how to connect to a simulator.
```

Comment:
Name Error: Find the Middle -> Find the center.  

Should Example names be in italic or quotes to set them off as an example name or is Title case sufficient?